README.md

```diff
@@ -1,34 +1,27 @@
-This repository contains a README file sample for Daytona Samples and the MIT License.
+# Sample Javascript/Nodejs
 
-It can be used as a template to create sample repositories that can be added into [Daytona](https://github.com/daytonaio/daytona).
+Thoughts is a web application built using NodeJs and EJS that allows user to share their thoughts.
 
-Once you finish your sample and it gets merged, you can open a PR in the Daytona repo and submit the sample into the [index file](https://github.com/daytonaio/daytona/blob/main/hack/samples/index.json).
-
-# Sample <LANGUAGE/FRAMEWORK>
-
-Sample description
-
----
-
-## 🚀 Getting Started  
+## 🚀 Getting Started
 
 ### Open Using Daytona  
 
 1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).  
+
 2. **Create the Workspace**:  
    ```bash  
-   daytona create <SAMPLE_REPO_URL> 
+   daytona create https://github.com/parikshitadhikari/thoughts-daytona.git 
```

Comment:
Please update the URL to https://github.com/daytonaio/sample-ejs-web-app

Reply:
done @Tpuljak